### DISCUSSION TITLE
Can we access environment variables in codegen.ts

### DISCUSSION THREAD

**Discussion by shivashankargalla** [dotansimha/graphql-code-generator#9558] [Q&A] [answered]
Can we access environment variables in  below codegen.ts file? **dev** should be dynamic in schema url.

import { CodegenConfig } from "@graphql-codegen/cli";

const config: CodegenConfig = {
  schema: `https://abc.**dev**.westus.aks.az.abc.com/hello/uapp/graphql`,
  documents: ["apps/event-flow/src/app/graphql/*.gql"],
  generates: {
    "apps/event-flow/src/app/graphql/generated/schema.ts": {
      plugins: [
        "typescript",
        "typescript-operations",
        "typescript-react-apollo",
      ],
    },
  },
  ignoreNoDocuments: true,
};

export default config;

**Comment by ardatan** [comment, marked as answer]
You can get environment variables from `process.env`
